pkg/repo/chartrepo_test.go

```diff
@@ -400,3 +400,19 @@ func TestResolveReferenceURL(t *testing.T) {
 		t.Errorf("%s", chartURL)
 	}
 }
+
+func TestEntry_URLWithTrailingSlash(t *testing.T) {
+	urlWithTrailingSlash := "http://someserver/something/"
+	e := Entry{URL: urlWithTrailingSlash}
+
+	if e.URLWithTrailingSlash() != urlWithTrailingSlash {
+		t.Errorf("Expected unchanged repository URL")
+	}
+
+	urlWithoutTrailingSlash := strings.TrimSuffix(urlWithTrailingSlash, "/")
+	e = Entry{URL: urlWithoutTrailingSlash}
+
+	if e.URLWithTrailingSlash() != urlWithTrailingSlash {
+		t.Errorf("Expected repository URL without trailing slash")
```

Comment:
Given the comparator above, shouldn't the error message report that it expected the repository URL _with_ the trailing slash?